_includes/footer.html

```diff
@@ -11,7 +11,7 @@
                 </ul>
                 <ul class="footer-links_secondary cf">
                     <li>
-                        <a href="http://cfpb.github.io/articles/introducing-cfpb-open-tech/">About</a>
+                        <a href="http://cfpb.github.io/cfpb-open-tech/">About</a>
```

Comment:
Would it work to make this a relative URL, or is that going to mess it up on GitHub pages?

Reply:
It should not be an issue, using a relative url. That update has been applied.

Reply:
Did the relative URL not get pushed?